menu.js

```diff
@@ -0,0 +1,35 @@
+const showMenu = document.getElementById('open-menu');
+showMenu.addEventListener('click', () => {
+  document.getElementById('nav-mobile').style.width = '100%';
+});
+
+const closebtn = document.getElementById('btn-close');
+closebtn.addEventListener('click', () => {
+  document.getElementById('nav-mobile').style.width = '0%';
+});
+
+const portfolioElement = document.getElementById('portfolio');
+portfolioElement.addEventListener('click', () => {
+  document.getElementById('nav-mobile').style.display = 'none';
+  // eslint-disable-next-line no-use-before-define
+  reloadBrowser();
+});
+
+const aboutElement = document.getElementById('about');
+aboutElement.addEventListener('click', () => {
+  document.getElementById('nav-mobile').style.display = 'none';
+  // eslint-disable-next-line no-use-before-define
+  reloadBrowser();
+});
+
+const contactElement = document.getElementById('contact');
+contactElement.addEventListener('click', () => {
+  document.getElementById('nav-mobile').style.display = 'none';
+  // eslint-disable-next-line no-use-before-define
+  reloadBrowser();
+});
+
+function reloadBrowser() {
+  // eslint-disable-next-line no-restricted-globals
+  location.reload();
+}
```

Comment:
- [ ] Kindly make sure you follow the design as per Figma for the menu from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, texts, margins).
- [ ] Kindly follow the background color as per the figma design.
- [ ] Kindly make sure that the menu options follow the same colour text , font size and spacing as per the design.
- [ ] Kindly make sure to follow the same color for the line after the options and for close button.
- [ ] Kindly make sure to display the white bottom section as shown in the attached video with a black line in the middle.

https://user-images.githubusercontent.com/70269507/173738546-7bc2e726-85c6-4263-bf5a-dbdd3581d42d.mov

- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. Currently, your about option doesn't take to the respective page. Fix it as shown below.

https://user-images.githubusercontent.com/70269507/173738989-ce719004-30a1-4a29-b395-01f9cbbd4a02.mov